backend/src/main/java/com/khnure/timetable/synchronizer/controller/CalendarController.java

```diff
@@ -0,0 +1,41 @@
+package com.khnure.timetable.synchronizer.controller;
+
+import com.khnure.timetable.synchronizer.dto.ScheduleDto;
+import com.khnure.timetable.synchronizer.service.NureScheduleService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Controller
+@RequiredArgsConstructor
+public class CalendarController {
+    private final NureScheduleService calendarService;
+
+    @ExceptionHandler
```

Comment:
Why weuse here the annotation? 

This annotation is used only for methods, which handles exception, which is thrown from the code.
See example [here of the correct usage of this annotation](https://www.baeldung.com/exception-handling-for-rest-with-spring#:~:text=This%20enables%20a%20mechanism%20that%20breaks%20away%20from%20the%20older%20MVC%20model%20and%20makes%20use%20of%20ResponseEntity%20along%20with%20the%20type%20safety%20and%20flexibility%20of%20%40ExceptionHandler%3A)